ajax/libs/switchy.js/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "switchy.js",
+  "filename": "switchy.min.js",
+  "title": "Switchy",
+  "description": "This is a tiny jQuery plugin which can be used as both a switch or a slider.",
+  "keywords": [
+    "switch",
+    "slider",
+    "select",
+    "form"
+  ],
+  "version": "0.1.1",
+  "author": {
+    "name": "lou",
+    "email": "louiscuny@gmail.com",
+    "url": "https://github.com/lou"
+  },
+  "homepage": "http://lou.github.com/switchy/",
+  "dependencies": {
+    "jquery": ">= 1.8"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/lou/switchy.git",
+    "basePath": "",
+    "files": [
+      "*.js",
+      "*.css"
```

Comment:
1. Use `"*.+(css|js|map)"` is better then `"*.js",` + "*.css"`
2. Should not use this kind of wildcard in non-dist folder/repo, try `"switchy*.+(css|js|map)"`